src/website/app/pages/Typography/examples.js

```diff
@@ -0,0 +1,150 @@
+
+const examples: TypeExamples = [
+  {
+    heading: (
+      <Heading level={2} id="long-form-text">
+        Long-form Text
+      </Heading>
+    ),
+    description: (
+      <Description variant="prose">
+        Default text and caption styles are intended for sections of content
+        that provide description and assistance. {' '}
+        <strong>Paragraph text</strong> is primarily used for content sections.{' '}
+        <strong>Caption Text</strong> is used to highlight or offset content,
+        such as inline help or when captioning of dashboard content.
+      </Description>
+    ),
+    examples: [
+      {
+        content: <Paragraph type="ui">Paragraph UI</Paragraph>,
+        value: '<p, li>',
+        color: 'color_gray_100',
+        font: 'Bold (14px/.875rem)'
+      },
+      {
+        content: (
+          <List>
+            <li>Alpha</li>
+            <li>Bravo</li>
+            <li>Charlie</li>
+          </List>
+        ),
+        // the design notes have <ol> here as well,
+        // but list-style will override any numbers shown on <ol> list items
+        value: '<ul, ol>',
+        color: 'color_gray_100',
+        font: 'Bold (14px)'
+      },
+      {
+        content: <Paragraph variant="prose">Paragraph (Long Form)</Paragraph>,
+        value: '<p.long>',
+        color: 'color_gray_100',
+        font: 'Regular (16px/1rem)'
+      },
+      {
+        content: <Paragraph variant="mouse">Paragraph (Mousetype)</Paragraph>,
+        value: '<p.mouse>',
+        color: 'color_gray_100',
+        font: 'Regular (11px/0.6875rem)'
+      },
+      {
+        content: <Heading level={5}>Caption - Strong</Heading>,
+        value: '<h5>',
+        color: 'color_gray_100',
+        font: 'Bold (14px/0.875rem)'
+      },
+      {
+        content: <Heading level={6}>Caption - Light</Heading>,
+        value: '<h6>',
+        color: 'color_gray_80',
+        font: 'Regular (14px/0.875rem)'
+      }
+    ]
+  },
+  {
+    heading: (
+      <Heading level={2} id="display-text">
+        Display Text
+      </Heading>
+    ),
+    description: (
+      <Description variant="prose">
+        <strong>Display text</strong> makes a bold statement, and is useful to
+        draw attention to different sections in an application view. Display
+        text is for short titles and section divisions. They should be used
+        sparingly – they aren’t for long-form content.{' '}
+      </Description>
+    ),
+    examples: [
+      {
+        content: <Heading level={1}>Page Title</Heading>,
+        value: '<h1>',
+        color: 'color_gray_100',
+        font: 'ExtraBold (34px/2.125rem)'
+      },
+      {
+        content: <Heading level={2}>Large Section Header</Heading>,
+        value: '<h2>',
+        color: 'color_gray_80',
+        font: 'SemiBold (28px/1.75rem)'
+      },
+      {
+        content: <Heading level={3}>Medium Section Header</Heading>,
+        value: '<h3>',
+        color: 'color_gray_80',
```

Comment:
The color used to actually style this heading does not match this value.

Reply:
this was an accidental reversion. I'm going to fix the `Heading.js` file in the #346 PR.